src/Components/DetailPages/DetailPages.js

```diff
@@ -0,0 +1,212 @@
+      fourthTag,
+      fifthTag,
+      firstPhoto,
+      secondPhoto,
+      thirdPhoto,
+      fourthPhoto,
+      fifthPhoto,
+    }=this.state.artworkDetails;
+    
+    const settings ={
+      //dots:true,
+      vertical:false,
+      infinite:true,
+      speed:500,
+      slidesToShow:5,
+      slidesToScroll:2,
+    };
+
+    return (
+      <body className="totalContainer">
+        {/* <Navbar /> */}
+        <div className="firstContainer">
+          <div className="titleBox">
+             <div className="title">
+                 <span>{title}</span>
+                 <img src="Images/ellipsis.png" alt="dot" />
+             </div>
+             <div className="titleWriter">
+                 <span className="titleby">by</span>
+                 <span className="writerName">{name}</span>
+                 <div className="date">{date}</div>
+                 <span className="view">조회 {viewNumber}</span>
+            </div>
+          </div>
+          <main>
+            <img className="artwork" src="Images/sea.jpeg" alt="sea" />
+            <div className="artworkDescribe">
+              <img src="Images/grafolweo.ico" alt="icon" />
+              <span className="artworkSelect">그라폴리오 '주목받는'에 선정된 작품입니다.</span>
+              <span className="date">{uploadDate}</span>
+            </div>
+            <div className="tag">
+              <span>{firstTag}</span>
+              <span>{secondTag}</span>
+              <span>{thirdTag}</span>
+              <span>{fourthTag}</span>
+              <span>{fifthTag}</span>
+            </div>
+            <span className="copyRight">Copyright © {name} All Rights Reserved.</span>   
+            <div className="likeBtn">
+              <ul>
+                <li>
+                  <img src="Images/hearteyes.png" alt="icon"/>
+                  <span className="text">좋아요</span>
+                  <span className="number">{likeNum}</span>
+                </li>
+                <li>
+                <img src="Images/starred.png" alt="icon"/>
+                  <span className="text">감동받았어요</span>
+                  <span className="number">{touchedNum}</span>
+                </li>
+                <li>
+                <img src="Images/sunglasses.png" alt="icon"/>
+                  <span className="text">사고 싶어요</span>
+                  <span className="number">{wantToBuyNum}</span>
+                </li>
+              </ul>
+            </div>
+            <div className="commentParts">
+              <div className="commentImg">
+               <img src="Images/starred.png" alt="icon"/>
+               <span>{likeBtnNum}</span>
+              </div>
+              <div className="comment">
+               <FaRegCommentDots className="commentDot" />
+               <span>{commentNum}</span>
+              </div>
+              <img className="share" src="Images/share.png" alt="icon"/>
+            </div>
+          </main>
+        </div>
+        <div className="emptyBar"></div>
```

Comment:
안에 content가 없는 태그는 selfclosing으로 작성해 주세요!

```suggestion
        <div className="emptyBar" />
```

Reply:
네